### ISSUE TITLE
individuals contribute:  add publications (PAPERS link)

### ISSUE THREAD

**Issue by julievano** [NCAR/hydrology#2]
for each publication, make a new file.  
(1) cp /docs/_posts/papers/_2000-01-01-template.md to YYYY-01-01-AAAA.md.  YYYY = year of publication, AAAA = first four letters of first author.  For multiple papers with the same AAAA and YYYY use 1,2,3 (follow examples in papers folder)
(2) fill out template as described.  You need at least one tag with the date.  You can also tag papers to multiple projects.

Re: what papers should be included: All papers done as part of your work with NCAR.  Prioritize those done as part of the projects highlighted on the website, but others can be included as well.

**Comment by gutmann** [comment]
Nice template system. 

Are the -01-01 valid to change?  e.g. can we have 2016-01-01-GUTM.md and 2016-05-01-GUTM.md 

If not, can that be fixed? For the most part this will take care of the multiple papers with matching author year (though we still need an additional numbering system for Martyn)

**Comment by julievano** [comment]
You could do different dates, although this makes it so they are no longer in reverse alphabetical order.   Right now it doesn't matter, but as we get more of Martyn's work into the system, the alphabetical ordering will come in handy.  Of course we can look at fixing all this to sort it differently, but I expect it'll be a more complicated fix as this is set up blog style so dates matter a lot.

**Comment by gutmann** [comment]
I would think it would just mean that first authors wouldn't be grouped by year, 2016-01 still comes before 2016-05 (?) or am I thinking about it wrong?

**Comment by julievano** [comment]
yes, although (if I'm understanding your question) if you have other authors in the mix, it's first sorted by date then by first author name, so it'd go: 
2016-01-06-GUTM.md
2016-01-01-WOOD.md
2016-01-01-GUTM.md

whereas, if you did 1,2, it'd be ordered:
2016-01-01-WOOD.md
2016-01-01-GUTM2.md
2016-01-01-GUTM1.md

Again, it's a simple work around.  Could be improved...

**Comment by nmizukami** [comment]
On web page,   Should we call "Publication" instead of "Paper"?

**Comment by julievano** [comment]
Good question.  I chose "Papers" as it seemed it would fit a broader audience (beyond the academic community).  Then too we can include other papers, not just publications - although I expect this will be the vast majority.
